pipelines/CI-presidio.yaml

```diff
@@ -1,73 +1,37 @@
+# CI pipeline for presidio. This pipeline triggers the build-test-publish template for every PR
+# and merge to master on https://github.com/microsoft/presidio.
+# Pipeline parameters:
+#   * PYPI_INDEX_URL - pypi artifact used to push the wheel.
```

Comment:
I think the comment should be revised a bit. 'artifact' is not so clear....

Reply:
done